test/performance/LatencyTestPublisher.cpp

```diff
@@ -676,11 +676,7 @@ bool LatencyTestPublisher::test(uint32_t datasize)
 
         lock.lock();
         data_cond_.wait_for(lock, std::chrono::seconds(1), [&]() { return data_count_ > 0; });
-
-        if(data_count_ > 0)
-        {
-            --data_count_;
-        }
```

Comment:
Be careful, I remember Latency teste supports more than 1 subscriber, so setting directly to 0 may not be correct.